docs/infrastructure.md

```diff
@@ -41,34 +42,40 @@ git clone https://github.com/neurobagel/api.git
 ### Set the environment variables
 Create a `.env` file in the root of the repository to house the environment variables used by the API-graph network.
 
-Below are all the possible Neurobagel environment variables that can be set in `.env`.
+!!! tip 
+    The `neurobagel/api` repo contains a [`.template-env`](https://github.com/neurobagel/api/blob/main/.template-env) file that you can copy and edit directly.
```

Comment:
```suggestion
    The `neurobagel/api` repo contains a [`.template-env`](https://github.com/neurobagel/api/blob/main/.template-env) to get you started. Copy and rename this file to `.env` and then edit it as needed.
```
I think this is essential enough to move into the plaintext (i.e. not just in the tip box).